types/chrome/index.d.ts

```diff
@@ -7184,25 +7184,50 @@ declare namespace chrome.storage {
+         * @return The `getBytesInUse` method provides its result via callback or returned as a `Promise` (MV3 only).
+         */
+        getBytesInUse(keys?: string | string[] | null): Promise<number>;
+        /**
+         * Gets the amount of space (in bytes) being used by one or more items.
+         * @param keys Optional. A single key or list of keys to get the total usage for. An empty list will return 0. Pass in null to get the total usage of all of storage.
          * @param callback Callback with the amount of space being used by storage, or on failure (in which case runtime.lastError will be set).
          * Parameter bytesInUse: Amount of space being used in storage, in bytes.
          */
         getBytesInUse(keys: string | string[] | null, callback: (bytesInUse: number) => void): void;
+        /**
+         * Removes all items from storage.
+         * @return The `clear` method provides its result via callback or returned as a `Promise` (MV3 only).
+         */
+        clear(): Promise<void>;
         /**
          * Removes all items from storage.
          * @param callback Optional.
          * Callback on success, or on failure (in which case runtime.lastError will be set).
          */
         clear(callback?: () => void): void;
+        /**
+         * Sets multiple items.
+         * @param items An object which gives each key/value pair to update storage with. Any other key/value pairs in storage will not be affected.
+         * Primitive values such as numbers will serialize as expected. Values with a typeof "object" and "function" will typically serialize to {}, with the exception of Array (serializes as expected), Date, and Regex (serialize using their String representation).
+         * @return The `set` method provides its result via callback or returned as a `Promise` (MV3 only).
+         */
+        set(items: { [key: string]: any }): Promise<void>;
         /**
          * Sets multiple items.
          * @param items An object which gives each key/value pair to update storage with. Any other key/value pairs in storage will not be affected.
          * Primitive values such as numbers will serialize as expected. Values with a typeof "object" and "function" will typically serialize to {}, with the exception of Array (serializes as expected), Date, and Regex (serialize using their String representation).
          * @param callback Optional.
          * Callback on success, or on failure (in which case runtime.lastError will be set).
          */
-        set(items: Object, callback?: () => void): void;
+        set(items: { [key: string]: any }, callback?: () => void): void;
+        /**
+         * Removes one or more items from storage.
+         * @param keys A single key or a list of keys for items to remove.
+         * @param callback Optional.
+         * @return The `remove` method provides its result via callback or returned as a `Promise` (MV3 only).
```

Comment:
Similar to the prior comments, I don't think it's useful to mention the callback here since it's not included on line 7230. I'd say `@return A void Promise`.